src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthenticationBackend.java

```diff
@@ -228,7 +230,7 @@ private static LdapEntry existsSearchingUntilFirstHit(final String user, Connect
                     baseSettings.get(ConfigConstants.LDAP_AUTHCZ_BASE, DEFAULT_USERBASE),
                     f,
                     SearchScope.SUBTREE,
-                    returnAttributes);
+                    returnAttributes, true);
```

Comment:
I think we should follow the setting consistantly in all cases, so this should still read from the setting